src/testing/rapid_tests.py

```diff
@@ -53,3 +80,25 @@ def _test_schools_and_educ_workers(states, contacts):
     too_long_since_last_test = states["cd_received_rapid_test"] <= -3
     to_test = eligible & too_long_since_last_test & has_educ_contacts
     return to_test
+
+
+def _give_rapid_tests_to_some_workers(states, contacts, share_workers_to_be_tested):
+    date = get_date(states)
+    work_cols = [col for col in contacts if col.startswith("work_")]
+    has_work_contacts = (contacts[work_cols] > 0).any(axis=1)
+
+    # starting 2021-04-26 every worker must be offered two tests per week
+    # source: https://bit.ly/2Qw4Md6
+    if date > pd.Timestamp("2021-04-26"):
```

Comment:
switch order of if/else from earlier dates to later dates. 

We need a more gradual transition from testing once per week to testing twice per week. Suggested test frequencies:
until 2020-04-07 (after easter vacation): 7
until 2020-04-12: 6
until 2020-04-17: 5
until 2020-04-22: 4 (this is the first switch to have a really strong effect)
until 2020-04-27: 3

Reply:
I think there is a mistake in this because we reach the frequency of 3 5 days before the law took effect. I implemented it such that we have equally spaced switches between Easter and the law such that we switch to a frequency of twice per week on the day the law took effect.

Reply:
I think there is a mistake in this. I think we want to **arrive** on the 27th at a frequency of every 3 days. This is how I implemented it in `f64e577`